packages/editor/src/components/post-slug/index.js

```diff
@@ -41,25 +42,19 @@ export class PostSlug extends Component {
 	}
 
 	render() {
-		const { instanceId } = this.props;
```

Comment:
I think we can remove `withInstanceId` HOC now https://github.com/WordPress/gutenberg/blob/adcad47e4d089696158f34a1ff9d84f5a5da0e47/packages/editor/src/components/post-slug/index.js#L84.

P.S. Unrelated, but it might be a good time to refactor this into a functional component.

Reply:
> I think we can remove withInstanceId HOC now

Right you are!

> P.S. Unrelated, but it might be a good time to refactor this into a functional component.

Eh, I don't want to spend too much time on this component that I don't think is even used 😅

Reply:
No worries. We have a general issue for refactoring into function components 😄